core/sass/calendar.scss

```diff
@@ -1,48 +1,138 @@
-$blue: 	#1947A3;
-
-section
-{
-    display: flex;
-    text-align: center;
-}
-
-table
-{
-    border-style: hidden;
-    border-collapse: collapse;
-    
-    -webkit-touch-callout: none;
-    -webkit-user-select: none;
-    -khtml-user-select: none;
-    -moz-user-select: none;
-    -ms-user-select: none;
-    user-select: none;
-    
-    cursor: default;
-}
-
-td
-{
-    border: 1px solid black;
-    padding: 10px;
-    width: 100px;
-    height: 15px;
-}
-
-#grid
-{
-    display: block;
-    background-color: white;
-    color: black;
-}
-
-td:hover
-{
-    background-color: $blue;
-    cursor: pointer;
-}
-
-th
-{
-    font-weight: bold;
-}
\ No newline at end of file
+$bg: #f6f3ea;
+$blue:  #1c305d;
+$red: red;
+$green: green;
+$highlight: #cac9c9;
+$disabledGray: #b3b1aa;
+
+html, body
+{
+  width: 100%;
+  height: 100%;
+}
+
+section
+{
+	position: relative;
+  text-align: center;
+  width: 800px;
+  height: 600px;
+  min-width: 300px;
+  min-height: 150px;
+}
+
+#weekView, #dayView
+{
+	position: relative;
+  text-align: inherit;
+  width: inherit;
+  height: inherit;
+  min-width: inherit;
+  min-height: inherit;
+}
+
+.hidden
+{
+  display: none;
+}
+
+#popup {
+  position: relative;
+  text-align: left;
+  width: 20%;
+  height: 25%;
+  background-color: rgba(255, 255, 255, 0.9);
+  //box-shadow: 5px 5px 5px #888888;
+	box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.5);
+}
+#popup:before {
+  content:"";
+  position: absolute;
+  right: 46%;
+  top: 100%;
+  width: 0;
+  height: 0;
+  border-left: 13px solid transparent;
+  border-right: 13px solid transparent;
+  border-top: 13px solid rgba(255, 255, 255, 0.9);
+}
+
+table
+{
+  border-style: hidden;
+  border-collapse: collapse;
+
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  cursor: default;
+
+  background-color: $bg;
+  width: inherit;
+  height: inherit;
+}
+
+td
+{
+  border: 1px solid black;
```

Comment:
This may interfere with the checkout table layout—maybe nest these under a specific table class, e.g.,

```scss
table.calendar {
   color: #BADDAD;
   td {
       border: solid 1px brown;
   }
}